### PR TITLE
[13.0] Set migration status for simple module without changes

### PR DESCRIPTION
**Pull request by kos94ok-3D**
Module list:
- attachment_indexation
- auth_ldap
- auth_oauth
- auth_password_policy
- base_address_city
- base_address_extended
- base_import
- base_import_module
- bus
- http_routing
- hw_blackbox_be
- hw_drivers
- hw_escpos
- hw_posbox_homepage
- hw_proxy
- hw_scale
- hw_scanner
- hw_screen
- pad
- web_diagram
- web_kanban_gauge
- web_tour

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by legalsylvain — COMMENTED** [review]
Regarding document module I think that there is a function in openupgrade to call. Dont' you think ?
Otherwise LGTM.

Code review / no test

**Comment by kos94ok-3D** [comment]
> Regarding document module I think that there is a function in openupgrade to call. Dont' you think ?
> Otherwise LGTM.
> 
> Code review / no test

Yeah, but modules were renamed and merged in a pre-migration script of a base module. Once for all. See PR #2205

**Review by legalsylvain — APPROVED** [review]
OK thanks !

File: odoo/openupgrade/doc/source/modules120-130.rst [hunk 1]
```diff
@@ -53,13 +53,13 @@ missing in the new release are marked with |del|.
 +----------------------------------------------+-------------------------------------------------+
 |association                                   |                                                 |
 +----------------------------------------------+-------------------------------------------------+
-| |new| attachment_indexation                  |                                                 |
+| |new| attachment_indexation                  | Done. Renamed of document                       |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules120-130.rst]:
```suggestion
| |new| attachment_indexation                  | Done. Renamed from document                     |
```

**Review by pedrobaeza — COMMENTED** [review]
FYI, OCA has unblocked the RfQ for migrating their instance to v13, so we are going to work from now on actively on OpenUpgrade v13.

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Pull request merged**